docker-compose.yml

```diff
@@ -0,0 +1,361 @@
+---
+version: '2.4'
+
+volumes:
+  builder-certs-ca: {}
+  builder-certs-client: {}
+  builder-data: {}
```

Comment:
oB doesn't have a builder so I guess we don't need these volumes, right?

Reply:
Yes, it has a pseudo-builder service (end of composition) for the e2e tests to run.